src/env.cc

```diff
@@ -1143,7 +1143,7 @@ CompileCacheEnableResult Environment::EnableCompileCache(
       compile_cache_handler_ = std::move(handler);
       AtExit(
           [](void* env) {
-            static_cast<Environment*>(env)->compile_cache_handler()->Persist();
+            static_cast<Environment*>(env)->FlushCompileCache(true);
```

Comment:
Nit: Something like this would be a lot more readable.

```suggestion
            static_cast<Environment*>(env)->FlushCompileCache(Mode::PERSIST_DESERIALIZED_CACHE);
```